code/modules/mob/living/brain/MMI.dm

```diff
@@ -42,6 +42,12 @@
 		if(brain)
 			to_chat(user, "<span class='warning'>There's already a brain in the MMI!</span>")
 			return
+		if(newbrain.decoy_override) //No MMI Test For You
+			to_chat(user, "<span class='warning'>There's a decent chance this is against the rules, admins have been notified and you're getting gibbed.</span>")
+			user.gib()
+			message_admins("[key_name(user)] was gibbed for trying to ling test someone with an MMI")
```

Comment:
```suggestion
			message_admins("[key_name(user)] tried to ling test someone with an MMI")
```

Reply:
I'm against using code to enforce policy, especially in this way, its a serious immersion-break when someone just explodes into a million if they put what they think is a brain into a machine; but you CAN totally add logging for stuff like this

Reply:
I don't disagree that it shouldn't happen in such an immersion breaking way way, but code should always enforce policy when possible. 

The no-ling testing policy is absolute - there is never a time when it is allowed, so it is a perfect candidate for something that should just be outright prevented by code.

Reply:
I'm with Ruko on this as far as round removing people who do stuff like this. MMI testing is an exploit resulting from a code oversight. 

That's why it became a "admin issue". While there is no 100% fix without completely reworking lings, an exploit is something that can be addressed with both code and admin work.

I'll take his option, if it means we make progress on getting this merged.